doc/1/getting-started/kotlin/index.md

```diff
@@ -0,0 +1,126 @@
+---
+code: false
+type: page
+title: Kotlin
+description: Kotlin Getting started
+order: 99
+---
+
+# Getting Started
+
+In this tutorial you will learn how to install the Kuzzle **Jvm SDK**.
+This page shows examples of scripts that **store** documents in Kuzzle, and of scripts that subcribe to real-time **notifications** for each new document created.
+
+::: success
+Before proceeding, please make sure your system meets the following requirements:
+
+- **Oracle JDK or OpenJDK** version 8 or higher ([OpenJDK installation instructions](https://openjdk.java.net/install/))
+- A running Kuzzle Server ([Kuzzle installation guide](/core/2/guides/essentials/installing-kuzzle))
+
+:::
+
+
+::: info
+Having trouble? Get in touch with us on [Discord](http://join.discord.kuzzle.io)!
+:::
+
+## Installation
+
+You can find the SDK JARs directly on [bintray](https://bintray.com/kuzzle/maven/kuzzle-sdk-jvm). Download and add it to your classpath.
+
+::: info
+The following examples are made to be executed without any IDE.
+If you're using Eclipse, IntelliJ or another Java IDE, you need to add the SDK as a project dependency in your classpath.
+:::
+
+### Installing on Android using gradle
+
+In your app build.gradle add the following line and sync.
+
+    implementation "io.kuzzle:kuzzle-sdk-jvm:1.0.0"
```

Comment:
This appear white on white background on the documentation https://deploy-preview-23--doc-sdk-jvm.netlify.app/getting-started/kotlin/#installing-on-android-using-gradle